transit_simulation/vehicle.py

```diff
@@ -0,0 +1,29 @@
+import uuid
+
+class Vehicle:
+    """
+    Base class for vehicle agents
+    """
+    def __init__(self, origin, destination):
+        self.id = uuid.uuid4()
+        self.current_location = origin
+        self.route = self.find_route(origin, destination)
+
+    def tick(self):
+        self.current_location = move()
+
+    def move(self):
+        raise NotImplementedError()
+    
+    def find_route(self, origin, destination):
```

Comment:
Do we really need find_route method also for buses? I thought we are arbitrarily assigning each bus agent a route

Reply:
Well that just means that the find_route method implementation will be different for buses, right?